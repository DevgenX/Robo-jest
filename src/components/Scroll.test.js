import { shallow } from "enzyme";
import Scroll from "./Scroll";

it("should render without errors", () => {
  expect(shallow(<Scroll />)).toMatchSnapshot();
});
