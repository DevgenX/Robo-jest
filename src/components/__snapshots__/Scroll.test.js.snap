// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without errors 1`] = `
<div
  style={
    Object {
      "border": "3px solid black",
      "height": "800px",
      "overflow": "scroll",
    }
  }
/>
`;
